pkg/agent/config/node_config.go

```diff
@@ -62,13 +61,17 @@ type NodeConfig struct {
 	Name string
 	// The name of the OpenVSwitch bridge antrea-agent uses.
 	OVSBridge string
+	// The name of the default tunnel interface. Defaults to "antrea-tun0", but can
+	// be overridden by the discovered tunnel interface name from the OVS bridge.
+	DefaultTunName string
```

Comment:
Out of curiosity. why is it called DefaultTunName? Are there non default tunnel interfaces?

Reply:
Right. We create extra tunnel ports to every remote Node when IPsec is enabled.